web/src/modules/i18n/containers/I18nProvider.js

```diff
@@ -88,9 +86,9 @@ export default ({ children }: PropsType) => {
     <I18nextProvider i18n={i18nextInstance}>
       <div
         data-testid={'direction'}
-        style={{
-          direction: config.isRTLLanguage(language) ? 'rtl' : 'ltr'
-        }}>
+        dir={config.isSupportedLanguage(language)
+          ? config.hasRTLScript(language) ? 'rtl' : 'ltr'
+          : undefined}>
```

Comment:
I don't really see a reason, why we want to remove the layout direction from the state. As you can see, this might very well be useful for some parts of the layout... or is there a replacement for it?
At least it should always match the direction that is set here by the I18nProvider.

Reply:
I removed it for simplicity as I thought there is no use of it anymore.

Reply:
I'll see whether I can refactor the only usage by using flexbox

Reply:
I reverted the changes :)